cmd/crc/cmd/start.go

```diff
@@ -133,3 +137,23 @@ func getPullSecretFileContent() (string, error) {
 
 	return pullsecret, nil
 }
+
+func checkIfNewVersionAvailable(noUpdateCheck bool) {
+	if noUpdateCheck {
+		return
+	}
+	newVersionAvailable, newVersion, err := version.NewVersionAvailable()
+	if err != nil {
+		logging.Debugf("Error checking if a new version is available: %v", err)
+		return
+	}
+	if newVersionAvailable {
+		logging.Infof("A new version (%s) has been published", newVersion)
+		userWantsToContinue := input.PromptUserForYesOrNo("Would you like to continue using the current version", false)
+		if !userWantsToContinue {
+			errors.ExitWithMessage(0, fmt.Sprintf("Please download the new version from %s", constants.DefaultPullSecretURL))
```

Comment:
forgot to mention, ExitWithMessage is a variadic function, so you can remove the Sprintf
```
errors.ExitWithMessage(0, "Please download the new version from %s", constants.DefaultPullSecretURL)
```
Now that I copied/pasted this, I notice `DefaultPullSecretURL` which should be renamed maybe?

Reply:
@cfergeau will change the name of this constant to `CrcLandingPageURL`.